pynestml/codegeneration/resources_spinnaker/common/iaf_psc_exp_template.c

```diff
@@ -0,0 +1,160 @@
+#include <iaf_psc_exp_template.h>
```

Comment:
Is this generated code, or the start of a new template? Generated code should not be part of the PR. If it is a template, could you rename it to something like ``@NEURON_NAME@.c.jinja2``?